edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/util/ChartModelProvider.java

```diff
@@ -63,10 +63,9 @@
 
 public class ChartModelProvider {
 
-  public static Chart createBarChart(String chartCaption, String xLabel, List<String> categoryCaptions, List<Double> values, Double average, Double q1, Double q3) {
+  public static Chart createBarChart(String chartCaption, String xLabel, List<String> categoryCaptions, List<Double> values, Double min, Double max, Double average, Double q1, Double q3) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)